packages/vm/package.json

```diff
@@ -19,6 +19,7 @@
   "contributors": [
     "Alex Beregszaszi <alex@rtfs.hu>"
   ],
+  "type": "commonjs",
```

Comment:
I would want to keep this for now. It is generally not yet 100% clear what setting is appropriate here and on the VM browser tests work better when setting here to `"commonjs"`(lol).